test/test_ccurl_interface.py

```diff
@@ -3,55 +3,318 @@
 
 from unittest import TestCase
 from pow import ccurl_interface
+from iota import Bundle, TransactionTrytes, TransactionHash
+from iota.transaction.validator import BundleValidator
+import time
+import copy
+
+from six import PY2
+
+if PY2:
+    from mock import MagicMock, patch
+else:
+    from unittest.mock import MagicMock, patch
+
+# Calculate time in milliseconds (for timestamp)
+get_current_ms = lambda : int(round(time.time() * 1000))
```

Comment:
💭 Wondering if `get_current_ms` should always return a constant value during unit tests.  Depending on how deterministic the underlying pow process is, if we can control all of the inputs, we could make the unit tests even stricter when they scrutinise the pow'd trytes.

Reply:
I decided to mock it away where it is needed :)